subprojects/android-test/ui-testing-core-app/src/androidTest/kotlin/com/avito/android/ui/test/Screen.kt

```diff
@@ -34,6 +34,9 @@ object Screen {
     val identicalCellsRecycler: IdenticalCellsRecyclerScreen
         get() = IdenticalCellsRecyclerScreen()
 
+    val keyboardScreen: KeyboardScreen
```

Comment:
A new instance creates on every access to this property. That really what we want? Or it can be wrapped into `lazy` for example?

Reply:
Do you see any problems with it? Does it heavy or has a state?